src/Swarm/Game/State.hs

```diff
@@ -626,6 +629,12 @@ scenarioToGameState scenario userSeed toRun g = do
           False -> id
           True -> const (fromList devices)
 
+  initialCaps = mconcat $ map (^. robotCapabilities) robotList
```

Comment:
Perhaps, when creativeMode is on, the initial list should be empty, so that there is still a sense of discovery?

Reply:
Hmm, no, when I'm playing creative mode I just want the game to stay out of my way and let me do whatever the heck I want (which includes showing me documentation for whatever the heck I want).

Reply:
Makes sense.